src/uniparc/components/sub-entry/SubEntryOverview.tsx

```diff
@@ -0,0 +1,150 @@
+import { memo } from 'react';
+import { Link } from 'react-router-dom';
+import { ExternalLink, InfoList, Loader } from 'franklin-sites';
+
+import TaxonomyView from '../../../shared/components/entry/TaxonomyView';
+
+import useDataApi from '../../../shared/hooks/useDataApi';
+
+import { getEntryPath } from '../../../app/config/urls';
+import apiUrls from '../../../shared/config/apiUrls/apiUrls';
+
+import { Namespace } from '../../../shared/types/namespaces';
+import { TabLocation as UniprotkbTabLocation } from '../../../uniprotkb/types/entry';
+import { UniParcSubEntryUIModel } from '../../adapters/uniParcSubEntryConverter';
+import { UniParcXRef } from '../../adapters/uniParcConverter';
+import { DataDBModel } from '../entry/XRefsSection';
+import EntrySection from '../../types/subEntry';
+
+type ExternalXrefLinkProps = { xref: UniParcXRef; dataDB: DataDBModel };
+
+const ExternalXrefLink = ({ xref, dataDB }: ExternalXrefLinkProps) => {
+  let { id } = xref;
+  if (!id || !xref.database) {
+    return null;
+  }
+  const template = dataDB.find(
+    ({ displayName }) => displayName === xref.database
+  )?.uriLink;
+  if (!template) {
+    return null;
+  }
+  // NOTE: exception for FusionGDB we need to remove the underscore number
+  if (xref.database === 'FusionGDB') {
+    id = id.replace(/_\d+$/, '');
+  }
+  return (
+    <ExternalLink url={template.replace('%id', id)}>
+      {xref.id}
+      {xref.chain && ` (chain ${xref.chain})`}
+    </ExternalLink>
+  );
+};
+
+type Props = {
+  data: UniParcSubEntryUIModel;
+};
+
+const SubEntryOverview = ({ data }: Props) => {
+  const dataDB = useDataApi<DataDBModel>(
+    apiUrls.configure.allDatabases(Namespace.uniparc)
+  );
+  const [proteomeId, component] =
```

Comment:
Maybe a `let [proteomeId, component] = [];` followed by ifs might be clearer to read

Reply:
Done.